### ISSUE TITLE
validation accuracy not match the published results in the paper

### ISSUE THREAD

**Issue by zlheui** [srebuffi/residual_adapters#3]
Hi,

I have ran the code and found some surprising results. For training from scratch, fine-tuning and parallel adapter training, the validation accuracy for the aircraft data set I get is 38%, 50%, 44%, however the published test accuracy are 57.10%, 60.34%, 64.21%. The results are lower than the published results.
For the UCF101 data set, the training from scratch, fine-tuning and parallel adapter training validation accuracy I get is 72%, 81%, 75%, however the published test accuracy are 43.27%, 51.20%, 50.94%. The results are higher than the published results.
Finally, the validation results I got for the cifar100 data set match the published results.
I simply ran the code in this github and did not make any changes. Although I am comparing the validation accuracy with the test accuracy, however the difference are quite large. Is there something wrong with the published results? Thanks.

**Comment by srebuffi** [comment]
Hello,
I thank you for your comment. Some participants of our Visual Decathlon Challenge also underlined that the validation set was not representative of the distribution of the test set for some datasets. Hence the differences you noted. We will fix that in a new version of the challenge.

**Comment by bkj** [comment]
Related question:  to reproduce the "Scratch" results in Table 1 of the paper, is this the correct command?
```
python train_new_task_from_scratch.py --dataset aircraft
```
(That gets about 38% accuracy on the validation dataset, as mentioned above.)

**Comment by pxpeng** [comment]
> Related question:  to reproduce the "Scratch" results in Table 1 of the paper, is this the correct command?
> python train_new_task_from_scratch.py --dataset aircraft
> 
> (That gets about 38% accuracy on the validation dataset, as mentioned above.)

I have met the same problem

**Comment by pxpeng** [comment]
> Hi,
> I have ran the code and found some surprising results. For training from scratch, fine-tuning and parallel adapter training, the validation accuracy for the aircraft data set I get is 38%, 50%, 44%, however the published test accuracy are 57.10%, 60.34%, 64.21%. The results are lower than the published results.
> For the UCF101 data set, the training from scratch, fine-tuning and parallel adapter training validation accuracy I get is 72%, 81%, 75%, however the published test accuracy are 43.27%, 51.20%, 50.94%. The results are higher than the published results.
> Finally, the validation results I got for the cifar100 data set match the published results.
> I simply ran the code in this github and did not make any changes. Although I am comparing the validation accuracy with the test accuracy, however the difference are quite large. Is there something wrong with the published results? Thanks.

Have you solved this problem?
